src/ImageSharp.Formats.Bmp/ConfigurationExtensions.cs

```diff
@@ -0,0 +1,29 @@
+﻿// <copyright file="ConfigurationExtensions.cs" company="James Jackson-South">
+// Copyright (c) James Jackson-South and contributors.
+// Licensed under the Apache License, Version 2.0.
+// </copyright>
+
+namespace ImageSharp
+{
+    using System;
+    using System.IO;
+
+    using Formats;
+
+    /// <summary>
+    /// Extension methods for the <see cref="Configuration"/> type.
+    /// </summary>
+    public static partial class ConfigurationExtensions
+    {
+        /// <summary>
+        /// Adds the BMP format.
+        /// </summary>
+        /// <param name="config">The config.</param>
+        /// <returns>The config</returns>
+        public static Configuration AddBmpFormat(this Configuration config)
```

Comment:
We don't need this nor the others.  The default behaviour is now for `Configuration.Default` to automagically load the types and I want to use the same syntax for adding all image formats.